packages/malloy/src/malloy.ts

```diff
@@ -785,6 +786,13 @@ export class Parse {
       (completion) => new DocumentCompletion(completion)
     );
   }
+
+  public helpContext(position: {
+    line: number;
+    character: number;
+  }): DocumentHelpContext | undefined {
+    return this.translator.helpContext(position).helpContext;
```

Comment:
I think it's important to have some tests for this behavior — it's pretty easy for the parser to change and totally break your walker without our realizing it. See `packages/malloy/src/lang/test/document-symbol-walker.spec.ts` for a rough example of how the tests could look.

Reply:
Yeah, that's a good idea.